### PR TITLE
[WIP] Design UI login page

### PR DESCRIPTION
**Pull request by duyhung0311**

### CODE REVIEW

File: frontend/src/feature/login/style.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    background-color: white    ;
+}
+.sign-in{
+    width: 100%;
+    height: auto;
+    font-size: 20px;
+    color: white;
+    background-color:#007c7e;
+    border-radius: 5px;
+    /* border-top: 40px; */
```
Comment (hiepvu0201) [inline frontend/src/feature/login/style.css:27]: can you remove this comment too?

File: frontend/src/feature/login/style.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    ;
+}
+.form-box{
+    width: 25%;
+    height: auto;
+    display:flex;
+    align-items: center;
+    justify-content: start;
+    border-radius: 10px;
+    /* box-shadow: 0px 0px 10px 2px gray; */
```
Comment (hiepvu0201) [inline frontend/src/feature/login/style.css:17]: can you remove this comment?

File: frontend/src/feature/login/style.css (hunk 1 above)
Comment (duyhung0311) [reply frontend/src/feature/login/style.css:27]: I think it can be removed

File: frontend/src/feature/login/style.css (hunk 2 above)
Comment (duyhung0311) [reply frontend/src/feature/login/style.css:17]: Yes, of course

**Pull request merged**